test/src/main/java/org/apache/accumulo/test/NamespacesIT.java

```diff
@@ -191,6 +192,18 @@ public void createTableInMissingNamespace() throws Exception {
     }
   }
 
+  @Test(expected = IllegalArgumentException.class)
+  public void createNamespaceWithNamespaceLengthLimit()
+      throws AccumuloException, AccumuloSecurityException, NamespaceExistsException {
+    StringBuilder namespaceBuilder = new StringBuilder();
+    for (int i = 0; i <= MAX_NAMESPACE_LEN; i++) {
+      namespaceBuilder.append('a');
+    }
+    String namespace = namespaceBuilder.toString();
+    c.namespaceOperations().create(namespace);
+    assertTrue(!c.namespaceOperations().exists(namespace));
```

Comment:
This code will never execute when the previous line throws an exception.

Reply:
The assertion line is not needed if we depend on the throw/not thrown status of the expected exception for testing. However, that does not really test for the existence of a table with an invalid name. So...I removed the expected notation and put the assertion in a try-catch block in the newest commit.